store-implementation/hbase-store/src/main/java/uk/gov/gchq/gaffer/hbasestore/utils/TableUtils.java

```diff
@@ -156,6 +156,10 @@ private static String getStorePropertiesPathString(final String[] args) {
         return args[2];
     }
 
+    private static String getFileGraphLibraryPathString(final String[] args) {
+        return args[3];
```

Comment:
You should check the length of args here. If args.length < 3 then return null;

Similarly for Accumulo but < 4.